guides/v2.2/cloud/env/restore-environment.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Restore an environment
+group: cloud-guide
+---
+
+If you encounter issues in your environment, but you do not have a [valid snapshot]({{ page.baseurl }}/cloud/project/project-webint-snap.html), you can try restoring your environment in on of the following ways:
+
+-  SSH
+-  Git
+
+## SSH
+
+If you have not created a snapshot but you can access the environment using SSH, follow these steps to restore your environment:
+
+-  Disable configuration management
+-  Uninstall Magento
+-  Reset the git branch
+
+Performing these steps:
+
+-  Returns your Magento installation to its original state by restoring the database, removing the deployment configuration, and clearing all directories under `var/`.
+-  Resets your git branch to a stable state in the past.
+
+### Disable configuration management
+
+You must disable [configuration management]({{ page.baseurl }}/cloud/live/sens-data-over.html) so that it does not automatically apply the previous configuration settings during deployment.
+
+To disable configuration management, make sure that your `app/etc/` directory does not contain the `config.php` file.
+
+To remove the configuration file:
+
+1. [SSH to your environment]({{ page.baseurl }}/cloud/env/environments-ssh.html#ssh).
+1. Remove the configuration file:
+   -  For Magento 2.2:
+
+      ```bash
+      rm app/etc/config.php
+      ```
+
+   -  For Magento 2.1:
+
+      ```bash
+      rm app/etc/config.local.php
+      ```
+
+### Uninstall Magento
+
+Uninstalling the Magento software drops and restores the database, removes the deployment configuration, and clears directories under `var/`.
```

Comment:
Wouldn't those be `var/` subdirectories?

Reply:
Indeed